GNUmakefile

```diff
@@ -6,7 +6,7 @@ GOTOOLS = \
 	github.com/elazarl/go-bindata-assetfs/go-bindata-assetfs@master \
 	github.com/hashicorp/go-bindata/go-bindata@master \
 	github.com/vektra/mockery/v2@latest \
-	github.com/golangci/golangci-lint/cmd/golangci-lint@v1.45.2 \
```

Comment:
You also have to edit `.circleci/config.yml` as the version number is duplicated there.

Reply:
https://github.com/hashicorp/consul/blob/bbcb1fa805c2eb02285d78faeb23996be035d32a/.circleci/config.yml#L190